mk/rules.mk

```diff
@@ -69,15 +69,15 @@ $(OBJS_DIR)/%.d: %.cc
 	@set -e; \
 	rm -f $@; \
 	echo "Finding dependencies of" $<; \
-	$(CXX) $(CDEPFLAGS) $(CPPFLAGS) $< $(DEP_FILTER) > $@.$$$$; \
+	$(CXX) $(CDEPFLAGS) $(INCLUDE_PATH) $(CPPFLAGS) $< $(DEP_FILTER) > $@.$$$$; \
 	$(DEP_REFORMAT) < $@.$$$$ | sed 's/: /: $$(wildcard /' | sed 's~\([A-Za-z]\)$$~\1)~' > $@; \
 	rm -f $@.$$$$
 
 $(OBJS_DIR)/%.d: %.c
 	@set -e; \
 	rm -f $@; \
 	echo "Finding dependencies of" $<; \
-	$(CC) $(CDEPFLAGS) $(CPPFLAGS) $< $(DEP_FILTER) > $@.$$$$; \
+	$(CC) $(CDEPFLAGS) $(INCLUDE_PATH) $(CPPFLAGS) $< $(DEP_FILTER) > $@.$$$$; \
```

Comment:
Adding `$(INCLUDE_PATH)` here will lead to it being duplicated on the command line as it's already appended to `$(CPPFLAGS)` in `stdmodule.mk`.  If this seems wrong I don't mind if you want to refactor it so that pre-processor flags and includes are rigorously treated as two separate things throughout the make config.  But please in a separate PR rather than this one.

Same on line 72 above.